tests/compose-rootfs/Containerfile

```diff
@@ -2,27 +2,34 @@
 FROM quay.io/centos/centos:stream10 as repos
 
 # You must run this build with `-v /path/to/rpm-ostree:/run/build/rpm-ostree:ro`
-FROM quay.io/fedora/fedora:41 as builder
+FROM quay.io/fedora/fedora-bootc:41 as builder
 RUN <<EORUN
 set -xeuo pipefail
-# Install our dependencies
-dnf -y install rpm-ostree selinux-policy-targeted sqlite
```

Comment:
This change is strictly speaking not related, but is convenient to avoid re-fetching these packages